kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/resolver/ResourceResolver.java

```diff
@@ -24,7 +27,20 @@ public static URL resolve(String location) {
 
                 return resource;
             } else {
-                return new URL(location);
+                // In case it is not a http, file nor a classpath protocol,
+                // we assume that this is plain text. We store it in a temporary
+                // file and return the URL to it.
+                File tmp = File.createTempFile("arquillian-cube", ".res");
```

Comment:
Can you move the creation of temporal file into a method like `createTemporalDefinition` which returns a `File` that is used later to create the URI? Just to have method more compact.